.drone.yml

```diff
@@ -127,13 +130,46 @@ pipeline:
       event: deployment
       environment: qa
 
-  deploy-to-prod:
+  deploy-to-qa-wcs:
     image: quay.io/ukhomeofficedigital/kd:v1.16.0
     environment:
+      - KUBE_NAMESPACE=wcs-qa
+      - ENVIRONMENT=qa
+      - KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+    secrets:
+      - hocs_casework_qa_wcs
+    commands:
+      - source version.txt
+      - echo $VERSION
+      - cd kube-hocs-casework
+      - ./deploy.sh
+    when:
+      event: deployment
+      environment: qa
+
+  deploy-to-prod-cs:
+    image: quay.io/ukhomeofficedigital/kd:v1.16.0
+    environment:
+      - KUBE_NAMESPACE=cs-prod
+      - ENVIRONMENT=prod
+      - KUBE_SERVER=https://kube-api-prod.prod.acp.homeoffice.gov.uk
+    secrets:
+      - hocs_casework_prod_cs
+    commands:
+      - cd kube-hocs-casework
+      - ./deploy.sh
+    when:
+      event: deployment
+      environment: prod
```

Comment:
This will mean when you deploy to prod you will deploy to both production environments; there won't be a way to deploy to just cs-prod or just wcs-prod.

Reply:
Well spotted, will modify for both qa and prod.